wiki/Ranking_Criteria/en.md

```diff
@@ -92,6 +92,7 @@ Glossary
 ### Guidelines
 
 - **Letterboxing and countdown options should be consistent between difficulties of a beatmapset.**
+- **Avoid directly re-using your own ranked beatmaps in other ranked beatmaps.** This is to avoid unnecessary bloating of Ranked content.
```

Comment:
```suggestion
- **Directly re-using your own ranked beatmaps to rank other beatmaps is discouraged.** This is to avoid unnecessary bloating of Ranked content.
```
"avoid" redundancy waffles

Reply:
adjusted (and made "ranked" --> "Ranked" everywhere for consistency)